doc/helpers/mediagallery.md

```diff
@@ -0,0 +1,149 @@
+---
+uid: Toolkit.Helpers.MediaGallery
+---
+# MediaGallery
+
+## Summary
+
+`MediaGallery` is a static class that allows interaction with the device's media gallery, providing methods to check access permissions and save media files.
+
+## Remarks
+
+This class is designed to work on iOS, Mac Catalyst and Android platforms, utilizing platform-specific implementations for its methods.
+
+The API allows setting the `targetFileName`, which **should ideally be unique** - otherwise the OS will either overwrite an existing file with the same name (Android behavior), or generate a new name instead (iOS behavior).
+
+## Methods
+
+### CheckAccessAsync
+
+```csharp
+public static async Task<bool> CheckAccessAsync()
+```
+
+#### Summary
+
+Checks the user's permission to access the device's gallery. Will trigger the permission request if not already granted.
```

Comment:
```suggestion
Checks the user's permission to access the device's gallery. It will trigger the permission request if it has not already been granted.
```